app/src/js/reducers/reconciliation-reports.js

```diff
@@ -1,82 +1,70 @@
 'use strict';
-import { set, del } from 'object-path';
-import assignDate from './assign-date';
 
+import assignDate from './utils/assign-date';
+import { createReducer } from '@reduxjs/toolkit';
 import {
   RECONCILIATION,
   RECONCILIATION_INFLIGHT,
   RECONCILIATION_ERROR,
-
   RECONCILIATIONS,
   RECONCILIATIONS_INFLIGHT,
   RECONCILIATIONS_ERROR,
-
   SEARCH_RECONCILIATIONS,
   CLEAR_RECONCILIATIONS_SEARCH,
-
   NEW_RECONCILIATION_INFLIGHT,
-  NEW_RECONCILIATION
+  NEW_RECONCILIATION,
 } from '../actions/types';
-import { createReducer } from '@reduxjs/toolkit';
 
 export const initialState = {
   list: {
     data: [],
     meta: {},
-    params: {}
+    params: {},
   },
   map: {},
   createReportInflight: false,
   search: {},
-  deleted: {}
+  deleted: {},
 };
 
 export default createReducer(initialState, {
   [RECONCILIATION]: (state, action) => {
-    const { id, data } = action;
-    set(state, ['map', id, 'inflight'], false);
```

Comment:
^^

Reply:
I'm pretty sure you're just going to point me to where this is getting set in the code or something. Because all of them are working.

Reply:
Yeah, as we chatted about before, nothing is checking _explicitly_ for the value `false`, but rather only for a "falsy" value, which is why simply removing the property works, but I am updating all places where you flagged this to explicitly set to `false` instead of removing the property.

Reply:
ok.  Hope it wasn't too painful for you.